tests/test_data.py

```diff
@@ -37,6 +56,11 @@ def get_all_test_cases():
 
 
 def get_multiple_input_test_cases():
+    """Get multiple input test cases.
+
+    :return: List of test cases
```

Comment:
If it's possible to explain what the function does in one line, then you can consider not writing the redundant `:return:`